package.json

```diff
@@ -55,6 +55,7 @@
     "react-i18next": "4.8.0",
     "react-native": "0.51.0",
     "react-native-background-timer": "2.0.0",
+    "react-native-calendar-events": "^1.4.3",
```

Comment:
I think it's common in this package.json to specify exact version.

Reply:
Very true. Fixed, thanks!